connexion/apis/aiohttp_api.py

```diff
@@ -171,7 +171,7 @@ def get_request(cls, req):
                      extra={'has_body': req.has_body, 'url': url})
 
         query = parse_qs(req.rel_url.query_string)
-        headers = {k.decode(): v.decode() for k, v in req.raw_headers}
+        headers = req.headers
```

Comment:
Does this work in aiohttp 2.3.10, and if not should we bump the required version in setup.py?

Reply:
Looks this feature is in 2.3.10.
@dutradda any reason to not use `.headers` for the CIMultiDictProxy?